README.md

```diff
@@ -10,31 +10,66 @@ provides for .NET projects.
+To use the plugin, open the `Conan.VisualStudio.sln` file using the Visual
+Studio 2017 and invoke the Run command on the developer machine (we aren't
+providing the installer packages yet). It will create an isolated Visual Studio
+environment and load the plugin.
 
 ### Configuration
 
-Enter "Tools → Options" menu and select "Conan" settings category. There you'll
-be able to set up the path to the Conan executable in your system (if it wasn't
-detected automatically).
+To use the Conan executable on your local system, the plugin needs to know where
+the Conan executable is. It will try to detect that automatically from your
+`PATH` environment variable, but you could set that manually for cases when the
+automatic detection doesn't work.
+
+To set the Conan executable location, enter "Tools → Options" menu and select
+"Conan" settings category.
 
 ![Settings window screenshot][screenshot-settings]
 
-### Installing the dependencies
+### Package installation
+
+The plugin will install the dependencies using the `conan install` command. To
+do that, ensure that your `conanfile.txt` is placed in the same directory as
+your project file, and then invoke the "Tools → Invoke AddConanDepends" menu
+command.
+
+!["Invoke AddConanDepends" menu item screenshot][screenshot-addconandepends]
+
+It will call `conan install --build missing --update` using the
+[`visual_studio_multi` generator][visual_studio_multi]. After that, you'll need
+to integrate the resulting property files into your Visual Studio project.
+
+### Integration with project
 
-After that, use the "Tools → Invoke AddConanDepends" command. It should invoke
-the `conan build` command for the project currently loaded into the IDE.
+The [`visual_studio_multi` generator][visual_studio_multi] creates the
+`conan/conanbuildinfo.props` property file that should be integrated into your
+`vcxproj` file. To do that, use the "Tools → Integrate into project" menu
+command. It will automatically add the corresponding `<Import>` item into the
+`vcxproj` file.
 
-### Using the tool window
+!["Integrate into project" menu item screenshot][screenshot-integrate]
+
+### Building without Visual Studio
+
+If you need to build the generated project without Visual Studio (e.g. on the
+build server machine), execute the following terminal commands:
+
+```console
+$ cd [directory with vcxproj]
+$ conan install . -g visual_studio_multi --install-folder ./conan -s compiler.version=15 --build missing --update
+$ msbuild [usual params here]
+```
 
-To show the plugin tool window, use the "View → Conan Package Management" menu
-item.
+After calling `conan install` that way, `msbuild` will be able to find all the
+dependencies, because it'll be able to use the Conan-generated
+`conan/conanbuildinfo.props` file.
 
 Build
 -----
 
-To build the project, either build it using Visual Studio or with MSBuild:
+To build the Conan Visual Studio plugin, either build it using Visual Studio or
+with MSBuild:
```

Comment:
I see three problems with this sentence:
1. "To build… build" is a tautology;
2. "using Visual Studio" but "with MSBuild";
3. not clear how to build with Visual Studio.

It seems like by "build using Visual Studio" you mean the recipe from "Usage" section above, but I'm not totally sure. I'm also not sure why the `msbuild` invocation in this section is different from the one in "Building without Visual Studio" section.

I don't know enough to suggest an alternative, but can't this be rewritten somehow?

Reply:
> It seems like by "build using Visual Studio" you mean the recipe from "Usage" section above, but I'm not totally sure. I'm also not sure why the msbuild invocation in this section is different from the one in "Building without Visual Studio" section.

This document describes two separate processes:

- using the plugin (to build **your** software)
- building the plugin itself

I'll look how to rewrite that, thanks.